stripe/src/main/java/com/stripe/android/AnalyticsRequestExecutor.kt

```diff
@@ -20,8 +18,7 @@ internal fun interface AnalyticsRequestExecutor {
         private val logger: Logger = Logger.noop()
     ) : AnalyticsRequestExecutor {
         private val connectionFactory = ConnectionFactory.Default()
-        private val job = SupervisorJob()
-        private val scope = CoroutineScope(Dispatchers.Default + job)
+        private val scope = CoroutineScope(Dispatchers.IO)
```

Comment:
Is this coroutine scope ever at risk of being cancelled? I don't think you can use a scope after it's been cancelled so something to be cautious of.

Reply:
It's not, but that's a good call. We should prefer creating a new scope for every operation.